modules/extensions-support/src/main/java/org/mule/module/extension/internal/introspection/ImmutableExtensionModel.java

```diff
@@ -67,6 +71,7 @@ protected ImmutableExtensionModel(String name,
 
         checkArgument(!StringUtils.isBlank(version), "version cannot be blank");
         this.version = version;
+        this.vendor = vendor;
```

Comment:
I see what you mean but I don't see such validation on the DefaultExtensionManager. I think that if vendor cannot be null, then it should not be possible to instantiate this class on an invalid state, specially when the containsExtension() method depends on it. I think that the validation should be moved here